medicines/api/src/main.rs

```diff
@@ -33,6 +33,10 @@ async fn healthz() -> impl actix_web::Responder {
     "OK"
 }
 
+async fn deploy_test() -> impl actix_web::Responder {
+    "I deployed this using AZ pipeline 💥 💫"
```

Comment:
I understand this was useful (and impressive) in testing, but do we still need it? It might be of value if it wasn't guilty of false-positives. If I ran this locally, I could see this message and it would be a lie!

Reply:
I mean to say: an endpoint which does declare some deployment or version information would be of value in future, for bug reproduction etc.

Reply:
I am in the same page of you @rjkerrison but is need it to pass QA as I described above on testing information of the PR, should be removed once graphQl server is done

Reply:
I removed